access.go

```diff
@@ -278,8 +284,11 @@ func (s *Server) handleRefreshTokenRequest(ctx context.Context, r *http.Request)
 		return nil, NewError(EInvalidClient, "request client id must be the same from previous token")
 	}
 
+	if previousRefreshToken.RedirectURI != req.RedirectURI {
+		return nil, NewError(EInvalidGrant, "invalid redirection uri for refresh token")
```

Comment:
can we say "request redirect uri must be the same as the one used to issue the refresh token"